generate-requirements-full.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
this file is different from that other PR we collaborated on a while back.

in order to not have redundancies in the requirements-core.txt file and so that we don't override the dependencies we installed and are pinned to in the datascience-python image this implementation is as follows.

* Pull the latest 7.x.y image of datascience-python
* run pip freeze to get a list of the existing dependencies on that container
* output the pip freeze dependencies to a tmp file
* append the requirements-core.txt file from this repo onto that tmp file
* now we use this tmp file as the input to generate requirements-full.txt

Reply:
i thought about pinning to datascience-python:7.0 or even datascience-python:7.0.0

I opted instead to pin to datascience-python:7 as this would reduce the overhead of updating dependencies. let me know what your thoughts are @jacksonlee-civis

Reply:
I think this general approach makes a lot of sense!